types/webpack-manifest-plugin/index.d.ts

```diff
@@ -1,92 +1,95 @@
-        /** Is required to run you app. Cannot be true if isChunk is false. */
-        isInitial: boolean;
-        isChunk: boolean;
-        /** Only available is isChunk is true. */
-        chunk?: Chunk;
-        isAsset: boolean;
-        /** Is required by a module. Cannot be true if isAsset is false. */
-        isModuleAsset: boolean;
-    }
-
-    interface Options {
-        /**
-         * The manifest filename in your output directory.
-         * Default: manifest.json
-         */
-        fileName?: string;
-
-        /**
-         * A path prefix that will be added to values of the manifest.
-         * Default: output.publicPath
-         */
-        publicPath?: string;
-
-        /**
-         * A path prefix for all keys. Useful for including your output path in the manifest.
-         */
-        basePath?: string;
-
-        /**
-         * If set to true will emit to build folder and memory in combination with webpack-dev-server
-         * Default: false
-         */
-        writeToFileEmit?: boolean;
-
-        /**
-         * A cache of key/value pairs to used to seed the manifest. This may include a set of custom key/value pairs to include in your manifest,
-         * or may be used to combine manifests across compilations in multi-compiler mode.
-         * To combine manifests, pass a shared seed object to each compiler's ManifestPlugin instance.
-         * Default: {}
-         */
-        seed?: object;
-
-        /**
-         * Filter out files.
-         */
-        filter?: (file: FileDescriptor) => boolean;
-
-        /**
-         * Modify files details before the manifest is created.
-         */
-        map?: (file: FileDescriptor) => FileDescriptor;
-
-        /**
-         * Sort files before they are passed to generate.
-         */
-        sort?: (a: FileDescriptor, b: FileDescriptor) => number;
-
-        /**
-         * Create the manifest. It can return anything as long as it's serialisable by JSON.stringify.
-         */
-        generate?: (seed: object, files: FileDescriptor[], entrypoints: { [key: string]: string[] }) => object;
-
-        /**
-         * Output manifest file in different format then json (i.e. yaml).
-         */
-        serialize?: (manifest: object) => string;
-    }
+export interface Options {
+    /**
+     * A path prefix for all keys. Useful for including your output path in the manifest.
+     * Default: ''
+     */
+    basePath?: string;
+
+    /**
+     * The manifest filename in your output directory.
+     * Default: 'manifest.json'
+     */
+    fileName?: string;
+
+    /**
+     * Filter out files which make up the manifest. Return true to keep the file, false to remove it.
+     */
+    filter?: (file: FileDescriptor) => boolean;
+
+    /**
+     * Create the manifest. It can return anything as long as it's serializable by JSON.stringify.
+     */
+    generate?: (seed: object, files: FileDescriptor[], entries: string[]) => object;
```

Comment:
@alex996 looks like the type of `entries` here is wrong - it hasn't changed since v2, and is still: `{ [key: string]: string[] }`.

Logging it out at compile time shows this to be the case too:
```ts
new WebpackManifestPlugin({
      fileName: "asset-manifest.json",
      generate: (seed, files, entrypoints) => {
        console.log(chalk.cyanBright("entrypoints", JSON.stringify(entrypoints, null, 2)));
		...
	  }
})
```

![image](https://user-images.githubusercontent.com/260431/102894088-3c5f4c80-446b-11eb-8b58-2a0704790e78.png)

Reply:
Looks like you are right. There might be a typo in the readme, as I following the [types mentioned there](https://github.com/shellscape/webpack-manifest-plugin/tree/3b2ed8af71c8e85189d056c9a9c8b1e0fc40f16e#generate).